modules/openid_auth/spec/requests/api_docs_reqeust_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
```

Comment:
Typo in file name 'reqeust'

Reply:
Thanks for catching that, I was in a hurry to rebuild the tests and should have caught this.